lib/kernel.js

```diff
@@ -101,7 +101,10 @@ module.exports = function(config, cb) {
 
   function stop() {
```

Comment:
in here you need to iterate through https://github.com/nearform/nscale-kernel/blob/master/lib/kernel.js#L131, and call `close` on all of them. These are the services. You can move all this logic into a _different_ `stopAllServices` method on `_containers`.